dsl/src/main/scala/com.crobox.clickhouse/dsl/language/ClickhouseTokenizerModule.scala

```diff
@@ -95,8 +94,16 @@ trait ClickhouseTokenizerModule extends TokenizerModule {
 
   private def tokenizeExpressionColumn(col: ExpressionColumn[_]): String =
     col match {
-      case agg: AggregateFunction[_]    => tokenizeAggregateFunction(agg)
-      case ArrayJoin(tableColumn)       => fast"arrayJoin(${tokenizeColumn(tableColumn)})"
+      case agg: AggregateFunction[_] => tokenizeAggregateFunction(agg)
+      case ArrayJoin(tableColumn)    => fast"arrayJoin(${tokenizeColumn(tableColumn)})"
+      case SplitString(tableColumn, separator) =>
+        if (separator.length == 1) {
+          fast"splitByChar(${StringQueryValue(separator)},${tokenizeColumn(tableColumn)})"
```

Comment:
Inconsisten spacing, triggering OCD :smiling_imp:

Reply:
It seems that that's the way scalafmt formats it, we could update the rules to try a fix.